Add sentry release github action

themarwhal opened the pull request:
Signed-off-by: Marie Bremner <marwhal@fb.com>

<!--
    Tag your PR title with the components that it touches.
    E.g. "[lte][agw] Changeset" or "[orc8r][docker] ..."
-->

## Summary
See doc: https://github.com/marketplace/actions/sentry-release
This will help us pinpoint which commits introduce specific errors.

<!-- Enumerate changes you made and why you made them -->

## Test Plan
test on master???
Will revert if it is broken
<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

themarwhal committed:
Add sentry release github action

Signed-off-by: Marie Bremner <marwhal@fb.com>

electronjoe reviewed (approved):
There are a couple YAMLLint comments to address! Otherwise LGTM.  The "truthy value should be..." one is unavoidable so I annotate it away (see other GH actions for example).

themarwhal commented:
> There are a couple YAMLLint comments to address! Otherwise LGTM. The "truthy value should be..." one is unavoidable so I annotate it away (see other GH actions for example).

Yeah wasn't sure how to fix that one, but I fixed the other space one.
Edit: fixed all lint
